src/opera/test/data/test_disp_input/compressed_slc_t087_185678_iw2_20180101_20180210.h5

```diff
@@ -0,0 +1 @@
+non-empty file
```

Comment:
Rather than have these test files permanently checked in, I would rather they be dynamically created/removed in the `setUp` and `tearDown` logic of the unit test module, similar to what we do for the other PGEs.